pkg/instrumentation/podmutator.go

```diff
@@ -20,8 +20,11 @@ import (
 	"fmt"
 	"strings"
 
+	"k8s.io/apimachinery/pkg/labels"
```

Comment:
should be merged with the block below

Reply:
I wonder why golangci-lint doesn't complain about this. We do have the `goimports` linter enabled.

Reply:
😂 I have used `goimports` and `lint` and have not found this issue, and now I have manually adjusted it.